docs/configuration.md

```diff
@@ -303,6 +303,77 @@ profiles:
        - '1.3.6.1.2.1.1.9.1.4.*'
 ```
 
+#### 4. Configure poller to return query with additional fields present
+
+User can provide **enricher** section to make poller enrich queries sent to Splunk by adding additional dimensions. There are two types of fields:
+1. **existingVarBinds**: this section updates query results with new fields calculated from the existing SNMP information.
+2. **additionalVarBinds**: this section updates query results with additional parameters defined below.
+
+##### Existing VarBinds
+This section works only for IF-MIB oid family.
+Every property of IF-MIB family can be extracted and added as an additional dimension to the query. For example, if we want to see the name and the index of the interface along with the basic query information,
+the **enricher** must be structured as following:
+```yaml
+enricher:
+  oidFamily:
+    IF-MIB:
+      existingVarBinds:
+        - ifIndex: 'interface_index'
+        - ifDescr: 'interface_desc'
+```
+From now on in every query for IF-MIB two new fields "interface_index" and "interface_desc" are visible:
```

Comment:
every query, but what type of query are we talking about? Maybe elaborate a little here